regolith/schemas.py

```diff
@@ -3807,7 +3868,31 @@
                                             f"Allowed values are {', '.join(MILESTONE_TYPE)}",
                              "required": False,
                              "type": "string",
-                             "eallowed": MILESTONE_TYPE}
+                             "eallowed": MILESTONE_TYPE},
+                    "end_date": {"description": "end date of milestone, yyyy-mm-dd",
+                                 "required": False,
+                                 "anyof_type": ["date", "string"]},
+                    "identifier": {"description": "label of milestone",
+                                   "required": False,
+                                   "type": "string"},
+                    "release_definition": {"description": "describes a release",
```

Comment:
I am not sure we are using this inside the milestones.  Maybe we want to cut it.   It is a bit of overkill.

Reply:
yes, I think that is best.  I don't remember what if anything I discussed with Zach, but I think it is too much structure and not useful, so cut.  Thanks.